src/zenml/integrations/gcp/image_builders/gcp_image_builder.py

```diff
@@ -177,11 +177,14 @@ def _configure_cloud_build(
         # Convert the docker_build_options dictionary to a list of strings
         docker_build_args = []
         for key, value in build_options.items():
+            option = f"--{key}"
```

Comment:
So the current implementation now works for the two args that we currently provide which is good, I'm pretty sure it's not generic and will break in some other cases.

I guess we just go with it for now? I don't have any idea which works generically other than not passing those arguments here and instead have them configure it as something separate in the `DockerSettings`.

Maybe something like `DockerSettings.image_builder_build_args` or something, but maybe not necessary for now.